src/sentry/event_manager.py

```diff
@@ -2764,8 +2763,12 @@ def _send_occurrence_to_platform(jobs: Sequence[Job], projects: ProjectsMapping)
                 detection_time=event.datetime,
                 level=job["level"],
             )
-
-            produce_occurrence_to_kafka(payload_type=PayloadType.OCCURRENCE, occurrence=occurrence)
+            try:
+                produce_occurrence_to_kafka(
+                    payload_type=PayloadType.OCCURRENCE, occurrence=occurrence
+                )
+            except KafkaException:
+                logger.exception("Failed to send occurrence to issue platform")
```

Comment:
I think we should just put this inside `produce_occurrence_to_kafka`

Reply:
i'm gonna merge as is and follow up to do that